src/Sulu/Bundle/CustomUrlBundle/Controller/CustomUrlController.php

```diff
@@ -95,7 +95,7 @@ public function postAction($webspaceKey, Request $request)
         $document = $this->get('sulu_custom_urls.manager')->create(
             $webspaceKey,
             $request->request->all(),
-            $this->getLocale($request)
+            $this->getRequestParameter($request, 'targetLocale', true)
```

Comment:
Does the `getLocale` method matter in this file? Then it should be overridden with returning null.